c7n/resources/vpc.py

```diff
@@ -703,6 +703,7 @@ def scan_groups(self):
                 ("sg-perm-refs", self.get_sg_refs),
                 ('lambdas', self.get_lambda_sgs),
                 ("launch-configs", self.get_launch_config_sgs),
+                ("ecs", self.get_ecs_sgs),
```

Comment:
let's rename to `ecs-periodic` there's potentially some expansion here to ecs task definitions overtime.

Reply:
Done!